database/postgres/postgres.go

```diff
@@ -182,15 +195,25 @@ func (p *Postgres) Unlock() error {
 	return nil
 }
 
+func getContext(statementTimeout time.Duration) (context.Context, context.CancelFunc) {
```

Comment:
Remove this method and just do all of the work inside `Run()`. This method doesn't add much value and the `if cancel != nil` conditional isn't very clean either.

Reply:
done